<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>931</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>在此插入学生信息：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>953</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_num"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_sex"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_age"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_birth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_phone"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_address"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>631</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_level">
      <property name="text">
       <string>按层次遍历排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_pre">
      <property name="text">
       <string>按前序遍历排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_in">
      <property name="text">
       <string>按中序遍历排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_post">
      <property name="text">
       <string>按后序遍历排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_getMinAge">
      <property name="text">
       <string>查询年龄最小的学生</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>941</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit_numsch"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_numsch">
      <property name="text">
       <string>搜索学号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_numdel">
      <property name="text">
       <string>删除学号</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_namesch"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_namesch">
      <property name="text">
       <string>姓名搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_namedel">
      <property name="text">
       <string>姓名删除</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_phosch"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_phosch">
      <property name="text">
       <string>电话搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_phodel">
      <property name="text">
       <string>电话删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>238</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="lineEdit_filename"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_selectfile">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
